scylla/src/transport/session_builder.rs

```diff
@@ -43,6 +45,36 @@ impl SessionBuilder {
         }
     }
 
+    #[cfg(test)]
+    pub async fn new_for_test() -> Session {
```

Comment:
I saw that this code was repeated all over the place, so this can be called in tests to setup a session with a test keyspace and table